infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -246,18 +246,18 @@ public void reset() {
         String d = serverContext.getDataStore().getLogDir();
         streamLog.close();
         if (d != null) {
-            Path dir = FileSystems.getDefault().getPath(d);
-            String prefixes[] = new String[]{"log"};
-
-            for (String pfx : prefixes) {
-                try (DirectoryStream<Path> stream =
-                             Files.newDirectoryStream(dir, pfx + "*")) {
-                    for (Path entry : stream) {
+            Path dir = FileSystems.getDefault().getPath(d + File.separator + "log");
+            try (DirectoryStream<Path> stream =
```

Comment:
This code is functionally different. In the previous patch, you delete everything in dir/log/*
but this patch deletes dir/* which has more files than just dir/log. Is that what you want?